src/Orleans.Persistence.Memory/Storage/MemoryStorageGrain.cs

```diff
@@ -13,28 +13,28 @@ namespace Orleans.Storage
     [KeepAlive]
     internal class MemoryStorageGrain : Grain, IMemoryStorageGrain
     {
-        private readonly Dictionary<(string, string), IGrainState> _store = new();
+        private readonly Dictionary<(string, string), object> _store = new(); // BPETIT TODO
```

Comment:
What's left TODO here? 😀

Reply:
Nothing really, I think that's the best compromise